### ISSUE TITLE
No support for aria-errormessage

### ISSUE THREAD

**Issue by DavidMacDonald** [FreedomScientific/standards-support#144]
### Summary

aria-errormessage not supported

Example:
1. http://davidmacd.com/test/display-none-aria-describedby.html
2.Tab to input. 

### Expected result

 Should hear error message

#### JAWS version and build number

2018 1805.33 ILM
 
#### Operating System and version
WIN 10

#### Browser and version:
Chrome 70.0.3538.102

**Comment by MichaelKetting** [comment]
I haven't used it yet due to lack of support but from the official description (https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage) I would say it is intended to work in the same manner as aria-labelledby and aria-describedby for annotating form fields (data edit widgets). 

To me, the difference with alert is that you may want to show a number of validation errors on a page after completing a postback. The user can then step through the invalid form fields and get the corresponding validation error announced  in a meaningful way.

**Comment by stevefaulkner** [comment]
filed internally as Bug 106336 - implement aria-errormessage

**Comment by stevefaulkner** [comment]
@BrettLewisVispero I cannot find any mention of `aria-errormessage` in the internal tracker, do you have any idea?

**Comment by stevefaulkner** [comment]
It looks like JAWS supports `aria-errormessage` in 2023.2307.37 (probably previously too)
test case https://cdpn.io/pen/debug/MWZggpM

- tab to first text box (`aria-errormessage`, `aria-invalid=false`) JAWS does not announce error message
- tab to 2nd text box (`aria-errormessage`, `aria-invalid=true`) JAWS announces "invalid entry, has error", then error message text
